_extras/snippets_library/UCL_sge/14/ls_dir_output.snip

```diff
@@ -1,14 +1,14 @@
 [output truncated] 
 
-conda-convert                       gio-querymodules        jupyter-run               python                tiff2rgba
-conda-develop                       glacier                 jupyter-serverextension   python3               tiffcmp
-conda-env                           glib-compile-resources  jupyter-troubleshoot      python3.6             tiffcp
-conda-index                         glib-compile-schemas    jupyter-trust             python3.6-config      tiffcrop
-conda-inspect                       glib-genmarshal         kill_instance             python3.6m            tiffdither
-conda-metapackage                   glib-gettextize         launch_instance           python3.6m-config     tiffdump
-conda-render                        glib-mkenums            lconvert                  python3-config        tiffinfo
-conda-server                        gobject-query           libpng16-config           pyuic5                tiffmedian
-conda-skeleton                      gresource               libpng-config             pyvenv                tiffset
-elbadmin                            hb-view                 patchelf                  rst2man.py
+ conda-convert                       gio-querymodules        jupyter-run               python                tiff2rgba
+ conda-develop                       glacier                 jupyter-serverextension   python3               tiffcmp
+ conda-env                           glib-compile-resources  jupyter-troubleshoot      python3.6             tiffcp
+ conda-index                         glib-compile-schemas    jupyter-trust             python3.6-config      tiffcrop
+ conda-inspect                       glib-genmarshal         kill_instance             python3.6m            tiffdither
+ conda-metapackage                   glib-gettextize         launch_instance           python3.6m-config     tiffdump
+ conda-render                        glib-mkenums            lconvert                  python3-config        tiffinfo
+ conda-server                        gobject-query           libpng16-config           pyuic5                tiffmedian
+ conda-skeleton                      gresource               libpng-config             pyvenv                tiffset
+ elbadmin                            hb-view                 patchelf                  rst2man.py
 
-[output truncated] 
+ [output truncated]
```

Comment:
Can you remove the first blank space on all this file? See how all starts with a space? remove them.

Reply:
Fixed in commit cd12006